### ISSUE TITLE
Mapillary android request, allow capture without login

### ISSUE THREAD

**Issue by nereocystis** [mapillary/mapillary_issues#2844]
### Basic information
* Platform:  `Android`
* Version: `3.117` 

### Desired behavior

Allow capture of images without logging in.
Uploads require logging in, of course.

Use case:
There are many locations in the world without network access. If my app is not logged in, then I cannot capture images with  the app.
Locations without network access probably have fewer visitors and it is less likely that anyone uploads images in these locations.

A second use case is when the app does not allow the user to log in. I am frequently experiencing this problem.

See 
https://github.com/mapillary/mapillary_issues/issues/2691

**Comment by katrinhumal** [comment]
@nereocystis Thanks for the suggestion! Normally, once you've logged in, you should not be automatically logged out of the app. Does this happen to you?

The standard use case is that you install the Mapillary app, log in, and stay logged in - which will allow you to access the capture screen also offline. If it doesn't then something is wrong and we need to investigate. If you get "kicked out" of the app (i.e. logged off without specifically doing so yourself) then something is wrong as well.

**Comment by mxa** [comment]
Albeit this issue should not be there as @katrinhumal said, it would be still a good precaution for cases where for whatever reason the app logs you out. Still happens from time to time.

**Comment by nereocystis** [comment]
@katrinhumal 
After a reboot of the phone, I was no longer logged in to Mapillary. Now I can't log in again.

Once I was logged in, I stayed logged in, until the reboot. I wasn't logged out automatically.

**Comment by katrinhumal** [comment]
@nereocystis Interesting - I'm not sure this is supposed to happen either. When I try on my own device, I stay logged in after a restart as well. @ovivoda how is this supposed to behave?

**Comment by nereocystis** [comment]
@katrinhumal After a reboot my openstreetmap login needs to be rebooted. Just checked again.

I haven't been able to log in using email in quite a while.

**Comment by ovivoda** [comment]
@katrinhumal @nereocystis The app should keep the user logged in even after restart

**Comment by mxa** [comment]
@ovivoda obviously, yes. But letting the user take pictures without being logged in would add a layer of safety. You still need the account for uploading, but if you are uploading you have the network access to also log in. I understand this would mean a lot of change to the inner workings of the app and It's probably too much work for solving a problem not many people will or should encounter in the first place, but it's still a valid feature request.

**Comment by katrinhumal** [comment]
@mxa Thank you! You have a very good point about the extra safety layer but we really appreciate your understanding of the effort/outcome ratio. We will, in any case, keep this in mind. If there wasn't a question of limited development resources, we'd surely see no reason for not implementing this right now.

@nereocystis Would you be able to turn on the "Save debug logs to file" in Settings > Developer, and after doing another device restart where you get logged out, send us the log file (from the left-hand menu)? Then @ovivoda can take a look at what's going wrong so that you don't stay logged in.

**Comment by mxa** [comment]
Not sure if this makes sense at all, but here is an idea: A completely separate bare-bones picture taking app that just stores picture sequences folders which can later be uploaded through the web app. Maybe this could be the start to the long awaited open source transition for mapillary.

**Comment by nereocystis** [comment]
@katrinhumal I finally sent the logs. I do have log in after reboot.

@mxa There are a few good open sources apps for starting points for taking the pictures. ChronoSnap takes images in intervals, though doesn't show the images. OpenCamera might be useful. So far, I haven't found any which offer the functionality of the Mapillary app.

**Comment by mxa** [comment]
@nereocystis getting off topic here... There is no android app that lets you take images based on GPS distance. My comment was in regard to the prospect by mapillary to open source the Android app. It would be good to see some progress there.
https://github.com/mapillary/mapillary_issues/issues/1791#issuecomment-207402410
https://github.com/mapillary/mapillary_issues/issues/1124#issuecomment-116312188

**Comment by mxa** [comment]
Related: https://github.com/mapillary/mapillary_issues/issues/1103

**Comment by katrinhumal** [comment]
_mapillary_issues is no longer an active repository.
Let’s continue discussing features in our forum! We will import this issue as topic to the Mapillary Forum in the coming weeks. We’ll notify here once it’s been done. See you there!_
